src/utils.js

```diff
@@ -42,7 +42,13 @@ export function get<T: any>(uri: string, options: any): Promise<T> {
             return;
         }
 
-        const opts = Object.assign({ json: true }, options);
+        const opts = Object.assign(
+            {
+                json: false,
+                token: null,
```

Comment:
Nah, a lot is moving around in mine. Go ahead & merge and I'll sort things out. Is my mess anyways 😜